workflows-scripts/inventory/exporting-inventory-search/README.md

```diff
@@ -0,0 +1,31 @@
+## Workflow steps:
+##### Workflow for inventory_saveUUIDsInstance.jmx
+1. Open Inventory App
+2. Select Instance tab in top left
+3. Select Query search in the drop down and type "keyword all "Boston" and source=FOLIO" in search field
+4. Under Actions menu click the option “Save instances UUIDs”
+
+## Modules required to be enabled as a pre-requisite to run JMeter script:
+##### Backend:
+- mod-inventory-storage-19.1.2
+- mod-inventory-14.1.2
+- mod-authtoken-2.4.0
+- mod-permissions-5.9.0
+- okapi-2.35.2
+##### Frontend:
+- folio_inventory-2.0.2
+
+- FOLIO release: FameFlower
+
+## Steps to run JMeter script:
+##### Using GUI:
+1. Open .jmx file in JMeter IDE
+2. Change credentials in jmeter-supported-data\credentials.csv 
+3. Change hostname in User Defined Variables
+4. Click to the play button
+
+##### Using non-GUI:
+Navigate to Apache bin folder and run script.
+For example:
+/apache-jmeter-5.2.1/bin                                              
+▶ sudo ./jmeter.sh -n -t [path_to_project]/perf-testing/workflows-scripts/item-search-filter/users_search.jmx -l [path_to_project]/perf-testing/workflows-scripts/item-search-filter/result.jtl
```

Comment:
After folder structure was modified, this path should be updated as well

Reply:
@javalkar-varun @mtraneis How about moving `Steps to run JMeter script:` to the README file int the root `perf-testing` folder or in the `workflows-scripts` folder? In order not to repeat these steps for each workflow

Reply:
Agreed. Thanks